modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,95 @@
+== Errata updates
+
+TBD
+
+[id="new-features-1-12-0_{context}"]
+== New features
+
+* With this update, the Argo CD CLI is supported and accessible as a productized and easily-installable RPM. The Argo CD CLI tool enables users to manage Red Hat OpenShift GitOps from a terminal. Using this tool, users can manage ArgoCD resources, such as Application, ApplicationSets, AppProjects, User accounts, and GPG keys from a client.
+
+[NOTE]
+====
+The `argocd` executable binary file is included in the archives and the RPMs.
+====
+
+[WARNING]
+==== 	
+The Argo CD CLI tool is a Technology Preview feature only.
+====
+
+* With this update, two new fields `.spec. applicationSet.sourceNamespaces` and `.spec. applicationSet.scmProviders` are introduced in ArgoCD CRD to support `ApplicationSet` in any namespaces. Administrators use it to define certain namespaces that manage ApplicationSet resources.
+
+* With this update, Argo CD server has the required permissions to manage `ApplicationSet` resources.
+
+* With this update, you can configure log levels such as "debug", "info", "warn", "error", "panic" and "fatal". The default log level set for the output is "info". Users can select an appropriate log level by adding the environment variable `LOG_LEVEL` in Red Hat OpenShift GitOps subscription's field, `spec.config.env`.
+
+.Example
+[source,yaml]
+----
+   yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: gitops-operator
+  namespace: openshift-gitops-operator
+spec:
+  ...
+  config:
+    env:
+    - name: LOG_LEVEL
+      value: "error"
+----
+
+* With this feature, Argo CD uses the wildcard values in `sourceNamespaces` to specify multiple namespaces or patterns of namespaces. To use this feature, specify the namespaces where Argo CD can manage applications in the Argo CD YAML with `spec.sourceNamespaces`. 
+
+.Example
+[source,yaml]
+----
+yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd-wildcard-pattern
+spec:
+  sourceNamespaces:
+    - app-team-*
+   - namespace-2
+----
+
+In this example, permissions are granted to namespaces that match the pattern `app-team-*`, such as `app-team-1`, `app-team-2`, and namespace-2, which does not use wildcard values.
+
+To grant permissions for all the namespaces on the Argo CD cluster using the `*` wildcard pattern, the following Argo CD CR configuration must be used:
+
+.Example
+[source,yaml]
+----
+yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd-all-namespaces
+spec:
+  sourceNamespaces:
+    - '*'
+----
+
+[id="fixed-issues-1-12-0_{context}"]
+== Fixed issues
+
+* Before this update, the notifications controller could not use `built-in` functions in notification templates to obtain information about applications due to an incorrect repo-server address. This update fixes the issue by initializing the notification controller to use the correct repo-server address.
```

Comment:
built-in is just a descriptor, I don't think it needs backticks

Reply:
Fixed.